### DISCUSSION TITLE
Communication hangs after some time (Sofar firmware V110022)

### DISCUSSION THREAD

**Discussion by Zwer2k** [wills106/homeassistant-solax-modbus#547] [General]
Since the update to firmware V110022, the communication via Modbus from my HYD 10KTL-3PH seems to hang after some time and only work again after switching off and on again from the Inverter. I use "Waveshare RS485 to RJ45 Ethernet Converter" and Solax integration for Home Assistant. Restarting Waveshare or HA does not help, always have to restart the IInverter. My brother is running a similar combination, only with a HYD 15KTL-3PH and is also having the failures. Does anyone else have these problems?

**Comment by wills106** [comment]
In the Wiki https://github.com/wills106/homeassistant-solax-modbus/wiki/Installation-Notes
Did you see this point:

> In "More Advanced Aettings...",
> 
> SolaX set Modbus Gateway Type to "Simple Modbus to TCP/IP mode"
> Sofar Solar & possibly Growatt set Modbus Gateway Type to "Auto query storage type"

Might help if not already set.

**Reply by Zwer2k** [reply]
Thank you for the answer.
This option is already set
